### DISCUSSION TITLE
nFormaterror is not defined when using Vite extension

### DISCUSSION THREAD

**Discussion by eduardojustos** [quasarframework/quasar#12048] [General - Components / Directives / etc]
I am using the Vite extension to use Quasar with tree handshake. The problem is that whenever I get a syntax error in a Vue file it says 

`[vite] Internal server error: nformatError is not defined
      at TransformContext.error (<dir>node_modules/vite/dist/node/chunks/dep-76613303.js:36765:13)
      at Object.transform (<dir>/node_modules/vite/dist/node/chunks/dep-76613303.js:36988:25)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
      at async doTransform <dir>/node_modules/vite/dist/node/chunks/dep-76613303.js:52049:29)
`

For example: I got in my script setup :

`const store = useStore();`

and then type something like 

`const store = useStore();something
`

I got this error. It is really annoying because I dont know where the error came from

Can anyone help me with this?

**Comment by hawkeye64** [comment]
I am not familiar with this error. Have you tried asking on the Vite discussions/Discord server?
